README.md

```diff
@@ -1,118 +1,119 @@
-
-
+فایل جدید randchunk پکت رو به شکل رندوم به 47 قسمت نامساوی تقسیم می‌کنه؛ -به روش سامورایی با تاخیر کم- یه کوچولو بهتر شده انگار.<br>
+رو ایرانسل همچنان مشکل داریم؛ ولی ترتیب اونم به امید خدا و یاری شما می‌دیم.<br>
+عزیزانی که تست می‌کنن از branch های دیگه مثل load balancer هم بازدید کنن.<br> 
+این محصول نهایی نیست! یه اسکریپت پایتونه جهت اثبات ادعا.<br>
+به نقطه‌ی پایدار که برسیم، برای کلاینت v2ray هم پیاده‌سازی می‌شه.<br>
+الان به متخصص حرفه‌ای شبکه و برنامه‌نویس زبان C نیاز داریم. برای تبادل اطلاعات، لطفاً در Discussions پیام بدید.<br>
 # internet for everyone or no one
-سلام گرم به همه دوستانی که برای حق اولیه و ابتدایی شهروندی ، برای دسترسی به اینترنت ، تلاش میکنند
+سلام گرم به همه دوستانی که برای حق اولیه و ابتدایی شهروندی، دسترسی به اینترنت، تلاش می‌کنند.
 <br>
-سلام به هیدیفای،باشسیز،سگارو،آی آر سی اف،پروژه امید،ثنایی،هگزا،وحید،صفا،اردشیر،ایمان،امین،حسین، یوتیوبرها و همه عزیزان دوست داشتنی
+سلام به هیدیفای، باشسیز، سگارو، آی آر سی اف، پروژه امید، ثنایی، هگزا، وحید، صفا، اردشیر، ایمان، امین، حسین، یوتیوبرها و همه‌ی عزیزان دوست داشتنی!
 <br><br>
-روش این پیج یک زخم عمیق بر پیکر  GFW  می گذارد که تا سالها سوزش آن در ماتحت فیلترچیان دنیا باقی خواهد ماند
+روش این پیج، یک زخم عمیق بر پیکر  GFW  می‌گذارد که تا سال‌ها سوزش آن در ماتحت فیلترچیان دنیا باقی خواهد ماند. 😁	
 <br>
 <br>
-خلاصه کار به فارسی:<br>
-روترهای gfw تلاش میکنند اما نمیتوانند همه packet های fragment را سرهم کنند زمانی که delay بین پکت ها باشد<br>
-چرا؟ چون کل ترافیک کشور ازشون عبور میکنه و براشون سخته و cache  محدود دارند و باید سریع باشند<br>
-از طرفی gfw نمیتونه پکت های فرگمنت رو reject کنه چون اولا fragmet جزو اصول شبکه هست<br>
-ثانیا در خیلی از نت های ضعیف packet ها تکه میشوند<br>
-در صورت reject کردن نت بسیاری از گوشی های قدیمی و خطوط ضعیف مختل میشه<br>
-همچنین در مسیر روترهای پرسرعت fragmentation اتفاق می افته<br>
-و اینو gfw میدونه بنابراین سعی میکنه اسمبل کنه و اگر نتونه عبور میده<br> 
-سرورها ولی موظف به سرهم کردن fragment ها هستند چون در پروتکل ip قید شده و سرشون به اندازه gfw شلوغ نیست<br>
-سرورهای کلودفلر به خوبی این کارو میکنن<br>
-باور کنید یا نکنید کار gfw ساختست<br>
-الان عمده ترافیک TLS هست و تنها با تحلیل SNI میتونه ترافیک TLS رو تفکیک کنه<br>
-و ما کار رو براش هزینه بر و پردازش بر میکنیم<br>
-یا باید کل cloudflare رو با همه سایت هاش ببنده و عملا نت جهانی رو قطع کنه<br>
- یا باید فرگمنت رو drop کنه که در هر صورت سیستم های خودشون هم دچار اختلال میشه<br>
-این سیستم تست شده و کار میکنه<br>
-و شما با domain فیلتر شده و با ip کثیف cloudflare میتوانید از gfw عبور کنید<br>
-با اندکی تنظیمات ، سرعت handshake اول هم بالا خواهد رفت<br>
-اینترنت برای همه یا برای هیچکس<br>
+خلاصه‌ی کار به فارسی:<br>
+روترهای gfw تلاش می‌کنن، اما نمی‌تونن همه‌ی packet های fragment را زمانی که delay بین پکت ها باشه سرهم کنن. <br>
+چرا؟ چون کل ترافیک کشور ازشون عبور می‌کنه و براشون سخته؛ همچنین، cache  محدود دارن و باید سریع باشن.<br>
+از طرفی، gfw نمی‌تونه پکت‌های فرگمنت رو reject کنه چون اولاً fragmet جزو اصول شبکه هست؛<br>
```

Comment:
In some places, the word fragment is not written correctly, like here, correct it and txt there is no problem.